exoctk/exoctk_app/app_exoctk.py

```diff
@@ -408,106 +408,101 @@ def contam_visibility():
-    #             visib_table = fh.getvalue()
-    #
-    #             # Format x axis
-    #             day0 = datetime.date(2019, 6, 1)
-    #             dtm = datetime.timedelta(days=367)
-    #             #vis_plot.x_range = Range1d(day0, day0 + dtm)
-    #
-    #             # Get scripts
-    #             vis_js = INLINE.render_js()
-    #             vis_css = INLINE.render_css()
-    #             vis_script, vis_div = components(vis_plot)
-    #
-    #             if request.form['submit'] == 'Calculate Visibility and Contamination':
-    #
-    #                 contamVars['contam'] = True
-    #
-    #                 # Make field simulation
-    #                 contam_cube = fs.sossFieldSim(contamVars['ra'],
-    #                                               contamVars['dec'],
-    #                                               binComp=contamVars['binComp'])
-    #                 contam_plot = cf.contam(contam_cube,
-    #                                         contamVars['tname'] or radec,
-    #                                         paRange=[int(contamVars['PAmin']),
-    #                                                  int(contamVars['PAmax'])],
-    #                                         badPA=pB, fig='bokeh')
-    #
-    #                 # Get scripts
-    #                 contam_js = INLINE.render_js()
-    #                 contam_css = INLINE.render_css()
-    #                 contam_script, contam_div = components(contam_plot)
-    #
-    #             else:
-    #
-    #                 contamVars['contam'] = False
-    #                 contam_script = contam_div = contam_js = contam_css = ''
-    #
-    #             return render_template('contam_visibility_results.html',
-    #                                    contamVars=contamVars, vis_plot=vis_div,
-    #                                    vis_table=visib_table,
-    #                                    vis_script=vis_script, vis_js=vis_js,
-    #                                    vis_css=vis_css, contam_plot=contam_div,
-    #                                    contam_script=contam_script,
-    #                                    contam_js=contam_js,
-    #                                    contam_css=contam_css)
-    #
-    #         except Exception as e:
-    #             err = 'The following error occurred: ' + str(e)
-    #             return render_template('groups_integrations_error.html', err=err)
+            # Calculate
+            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+
+            # Make plot
+            TOOLS = 'crosshair, reset, hover, save'
+            fig = figure(tools=TOOLS, plot_width=800, plot_height=400, x_axis_type='datetime',
+                         title=form.targname.data or ', '.join([form.ra.data, form.dec.data]))
+            fh = StringIO()
+            table.write(fh, format='ascii')
+            visib_table = fh.getvalue()
```

Comment:
this was edited on the master version of ExoCTK that doesn't have my latest updates on the output visibility table (https://github.com/ExoCTK/exoctk/pull/242) - if my PR gets merged, would that overwrite the work that was done here?

Reply:
Let's merge this then resolve the conflicts in #242. It shouldn't be a problem.